src/components/constraints/ConstraintListItem.svelte

```diff
@@ -108,37 +263,82 @@
             <StatusBadge status={Status.Unchecked} />
           </span>
         {/if}
-        <button
-          use:tooltip={{ content: visible ? 'Hide' : 'Show', placement: 'top' }}
-          class="st-button icon"
-          on:click|stopPropagation={() => dispatch('toggleVisibility', { id: constraint.id, visible: !visible })}
-        >
-          {#if visible}
-            <VisibleShowIcon />
-          {:else}
-            <VisibleHideIcon />
+        <div class="order-container">
+          <input
+            bind:this={orderInput}
+            bind:value={order}
+            class="st-input"
+            min="0"
+            style:width="68px"
+            type="number"
+            on:change={onUpdateOrder}
+            on:keydown={onKeyDown}
+            use:permissionHandler={{
+              hasPermission: hasEditPermission,
+              permissionError: editPermissionError,
+            }}
+          />
+          {#if hasEditPermission}
+            <div class="order-buttons">
+              <button
+                use:tooltip={{ content: 'Increase order', placement: 'top' }}
+                class="st-button tertiary up-button"
+                class:hidden={upButtonHidden}
+                tabindex={upButtonHidden ? -1 : 0}
+                on:click={onIncreaseOrder}
+              >
+                <CaretUpFillIcon />
+              </button>
+              <button
+                use:tooltip={{ content: 'Decrease order', placement: 'top' }}
+                class="st-button tertiary down-button"
+                on:click={onDecreaseOrder}
+              >
+                <CaretDownFillIcon />
+              </button>
+            </div>
           {/if}
-        </button>
-        <select
-          class="st-select"
-          value={constraintPlanSpec.constraint_revision}
-          on:change={onUpdateRevision}
-          on:click|stopPropagation
-          use:permissionHandler={{
-            hasPermission: hasEditPermission,
-            permissionError: readOnly
-              ? PlanStatusMessages.READ_ONLY
-              : 'You do not have permission to edit plan constraints',
-          }}
-        >
-          <option value={null}>Always use latest</option>
-          {#each revisions as revision, index}
-            <option value={revision}>{revision}{index === 0 ? ' (Latest)' : ''}</option>
-          {/each}
-        </select>
+          <button
```

Comment:
<img width="187" alt="image" src="https://github.com/user-attachments/assets/8b74c08d-2469-4580-af73-f2dcc05d3cf4" />
Can you add some horizontal padding to the visibility button?